docs/user/filesystem/commandline.rst

```diff
@@ -133,8 +131,50 @@ Note that in our example January 1st and 2nd both do not have all four quadrants
 dataset for that day didn't have any data in that region of the world. If additional data were ingested, the directory
 and a corresponding file would be created.
 
-``update-metadata``
+``manage-metadata``
```

Comment:
I wonder if this should be shortened to 'metadata'.  

Also, is there any reason that one might want to get a portion of the metadata?  I suppose I'm wondering what other metadata commands might be useful.

Reply:
for consistency, we've been trying to keep command names as 'verb-noun' (at least I have...). Do the existing commands for listing partitions, files, etc cover your question? I didn't modify those.